Apps/phsentinelone/sentinelone_connector.py

```diff
@@ -185,15 +186,26 @@ def _handle_block_hash(self, param):
         hash = param['hash']
         description = param['description']
         os_family = param['os_family']
-        summary = action_result.update_summary({})
-        summary['hash'] = hash
-        summary['description'] = description
-        header = self.HEADER
-        header["Authorization"] = "APIToken %s" % self.token
-        params = {"value": hash, "type": "black_hash"}
-        ret_val, response = self._make_rest_call('/web/api/v2.1/restrictions', action_result, headers=header, params=params)
-        if phantom.is_fail(ret_val):
-            return action_result.get_status()
+        try:
+            site_val = self._get_site_id(action_result)
+        except Exception:
+            return action_result.set_status(phantom.APP_ERROR, "Did not get proper response from the server")
+        self.save_progress('Agent query: {}'.format(site_val))
+        if site_val == '0':
+            return action_result.set_status(phantom.APP_ERROR, "Endpoint not found")
+        elif site_val == '99':
```

Comment:
Are we going to loop over multiple sites in case there exists?

Reply:
There is only one site so, we are not looping over the site.